assets/helpers/abtest.js

```diff
@@ -89,14 +93,22 @@ function getMvtId(): number {
 
 function getLocalStorageParticipation(): Participations {
 
-  const abTests = localStorage.getItem('gu.support.abTests');
+  let abTests = null;
+
+  if (localStorageExists) {
```

Comment:
Instead of having a function which tests the localStorage, Is it possible to have a function that set the item if locaStorage is available? In that way, we won't have to be asking if localStorage is available every time and we will save 2 lines of code for every use of localStorage.